CMakeLists.txt

```diff
@@ -125,6 +125,26 @@ set(SCAP_WORKBENCH_LINK_LIBRARIES
     Qt5::Widgets Qt5::XmlPatterns
     ${OPENSCAP_LIBRARIES})
 
+# ---------- RPATHS for linking
+
+# see https://gitlab.kitware.com/cmake/community/wikis/doc/cmake/RPATH-handling
+
+if (APPLE)
```

Comment:
let's make this universal and not apple only please

Reply:
@mpreisler done

Reply:
wait, have you reworked and tested it or just removed the if?

Reply:
@mpreisler unless you are seeing something different, compiled and ran it locally and appears to be working.